README.md

```diff
@@ -418,16 +418,18 @@ type Position = {
 
 type Output = {
   total: bigint
+  duration: number | null
   withdrawable: bigint
   positions: Position[]
 }
 ```
 
-| Name | Description |
-|------|-------------|
-| `positions` | Queue positions |
-| `total` | Total withdrawal amount (in ETH) |
-| `withdrawable` | Amount available for withdrawal (in ETH) |
+| Name | Description                                                                                                      |
+|------|------------------------------------------------------------------------------------------------------------------|
+| `positions` | Queue positions                                                                                                  |
+| `total` | Total withdrawal amount (in ETH)                                                                                 |
+| `duration` | Queue duration time (in timestamps), but if the value is "null" it means that the time is still being calculated |
+| `withdrawable` | Amount available for withdrawal (in ETH)                                                                         |
```

Comment:
Assets available for withdrawal (e.g. ETH)

Reply:
https://github.com/stakewise/v3-sdk/pull/90/commits/b3d165a383d328069f8366b2174e6d9dd59c8497